### DISCUSSION TITLE
为啥undo_log表的的数据事务回滚之后不能被删除

### DISCUSSION THREAD

**Discussion by xujiliang** [apache/incubator-seata#5090] [Q&A] [answered]
事务回滚之后，undo_log表的数据（status = 0）没有被删除，该表积压大量数据，tc一直在后台重试，导致数据库表空间急剧增加。请问什么原因导致undo_log数据没有被删除，如果解决该问题？

**Comment by wangliang181230** [comment]
回滚成功的话，应该只有delete的SQL执行失败，才会出现这种情况吧。
默认情况下，7天后也会被删除。

这个功能出现BUG的概率还是比较低的。

**Comment by funky-eyes** [comment]
说明没有回滚成功，被脏写了，仔细看rm侧被重试的日志

**Comment by funky-eyes** [comment, marked as answer]
说明没有回滚成功，被脏写了，仔细看rm侧被重试的日志
